fv3config/_datastore.py

```
@@ -181,6 +181,45 @@ def get_microphysics_name(config):
 def get_field_table_filename(config):
     """Get field_table filename given configuration dictionary
 
+    Args:
+        config (dict): a configuration dictionary
+
+    Returns:
+        str: field_table filename
+
+    Raises:
+        ConfigError
+    """
+    field_table = config.get("field_table", None)
+    if field_table is None:
+        field_table_dir = os.path.join(DATA_DIR, "field_table")
+    elif filesystem.isabs(field_table) and filesystem.get_fs(field_table).exists(
+        field_table
```

Don't check `exists` here. If the field table does not exist, we want the program to error and tell the user they've given an invalid option, instead of silently failing and selecting the default option.

---

Should add a test on this function that makes sure it raises `ConfigError` (in the else block below) if it's given a non-existent absolute path.

---

Happy to add another test; I think this behaves as desired in that case, however, unless I'm not understanding.  Agree with your earlier comment though that this function could be broken up to make things clearer!